setup.py

```diff
@@ -28,6 +26,7 @@
     'google-api-python-client>=1.6',
     'intervaltree>=2.1.0,<2.2.0',
     'pyvcf<0.7.0',
+    'google-nucleus>=0.2.0',
```

Comment:
For now, let's actually pin this to `0.2.0` as they may release a new version, which may break our unit tests.